feat(app): PRTL-295 update error messages (#1095)

cy opened the pull request:
- remove 'accept certificate' message
- always link gdc support in error modals
Closes #295

alex-wilmer commented on app/scripts/app.ts:
```diff
@@ -167,19 +167,7 @@ function appRun(
-      container: "#notification",
-      classes: "alert-danger"
-    });
+    console.log('error retriving status');
```
Do we need this? If so spellcheck?

alex-wilmer commented on app/scripts/components/downloader/downloader.directive.ts:
```diff
@@ -76,7 +78,7 @@ module ngApp.components.downloader.directive {
         <a data-ng-click="cancelDownload()"><i class="fa fa-times-circle-o"></i> Cancel Download</a>';
 
-      const detailedMessage = '<span>The download preparation can take time due to different factors (total file size, number of files, or number of concurrent users). \
+      const detailedMessage = '<span>The download preparation can take time due to a variety of factors (total file size, number of files, or number of concurrent users). \
```
Looks like we have multiline strings now right? if so can we use that and get rid of the backslashes?

cy commented:
@alex-wilmer addressed your comments. Also I actually found a file that 404s (https://gdc-portal.nci.nih.gov/legacy-archive/files/ed935a40-3e5e-49ca-ac8c-93e9faf0e79b) decided it's strange to return the same message as internal server error; so for 500s displaying the message that's in the spec for failed API calls (The connection to the GDC API failed. Please try to refresh the page or contact GDC Support if the problem persists.), otherwise still show the message passed by the api.
